Code/Editor/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ add_subdirectory(Plugins)
 
 if(CARBONATED_ENGINE_ADDONS)
 set(azcommonaddon_dependency Legacy::CryCommonAddon)
+set_property(GLOBAL APPEND PROPERTY LY_EDITOR_PLUGINS $<TARGET_FILE_NAME:Legacy::MadWorldEditorPlugin>)
```

Comment:
No, this cannot exist in the engine.
Please work with Amazon to figure out the "right way" to add this.

Reply:
Moved this directive into the beginning of the file "\\o3de-gruber\CMakeLists.txt" 
into Gruber PR: https://github.com/carbonated-dev/o3de-gruber/pull/565